resources/dd-ix.net.zone

```diff
@@ -0,0 +1,60 @@
+; Check me with (abs. path required....)
+; named-checkzone dd-ix.net /.../resources/dd-ix.net.zone
+
+$TTL    1h         ; default TTL for zone
+$ORIGIN dd-ix.net. ; base domain-name
+
+@  3600  IN  SOA  ns.dd-ix.net  noc.dd-ix.net.  (
+        2020080302  ;Serial
+        7200        ;Refresh
+        3600        ;Retry
+        1209600     ;Expire
+        3600        ;Negative response caching TTL
+)
+
+;; NS Records
+@  86400   IN  NS  ans-01.ibh.de.
+@  86400   IN  NS  ans-02.ibh.net.
+@  86400   IN  NS  ans-03.ibh.de.
+@  86400   IN  NS  ans-04.ibh.services.
+@  86400   IN  NS  ans-05.ibh.net.
+
+;; A Records
+@    600  IN  A  212.111.245.178
+rpx  600  IN  A  212.111.245.178
+
+ns   600  IN  A  212.111.245.179
+
+;; CNAME Records
+www      IN  CNAME  rpx
+content  IN  CNAME  rpx
+
+auth     IN  CNAME  rpx
+cloud    IN  CNAME  rpx
+wiki     IN  CNAME  rpx
+dcim     IN  CNAME  rpx
+lists    IN  CNAME  rpx
+vault    IN  CNAME  rpx
+
+;; BACKUP
+*        IN  CNAME  @
+
+autoconfig       IN  CNAME  autoconfig.migadu.com.
+key1._domainkey  IN  CNAME  key1.dd-ix.net._domainkey.migadu.com.
+key2._domainkey  IN  CNAME  key2.dd-ix.net._domainkey.migadu.com.
+key3._domainkey  IN  CNAME  key3.dd-ix.net._domainkey.migadu.com.
+
+;; MX Records
+@  IN  MX  20  aspmx2.migadu.com.
```

Comment:
I prefer to keep the zone content sorted by the owner name => move the `@` up to the SOA RR